ui/page/wallet/wallet_settings_page.go

```diff
@@ -373,26 +430,25 @@ func (pg *SettingsPage) changeSpendingPasswordModal() {
 			}
 			pg.wallet.LockWallet()
 
-			// change password
-			newSpendingPasswordModal := modal.NewCreatePasswordModal(pg.Load).
-				Title(values.String(values.StrChangeSpendingPass)).
-				EnableName(false).
-				PasswordHint(values.String(values.StrNewSpendingPassword)).
-				ConfirmPasswordHint(values.String(values.StrConfirmNewSpendingPassword)).
-				SetPositiveButtonCallback(func(walletName, newPassword string, m *modal.CreatePasswordModal) bool {
-					err := pg.wallet.ChangePrivatePassphraseForWallet(password,
-						newPassword, sharedW.PassphraseTypePass)
+			currentPassword = password
+			if pg.AssetsManager.DexcReady() {
```

Comment:
Just seeing `mgr.DexcReady` for the first time. Wondered why dexc would be initialialized but not be ready. Turns out the method checks if it is initialized, not if it is ready. Please rename mgr.DexcReady to mgr.DexcInitialized.